### PR TITLE
Arkade get: removed sudo suggestion and changed suggested path for Git Bash users.

### LINKED ISSUE #169: [Git Bash] Arkade get OSM path errors
## Expected Behaviour

On Windows Git Bash, when running `ark get osm`, the last two commands should be able to run on Git Bash with errors.
The commands should be adapted as follow:

```bash
chmod +x C:/Users/corsair/AppData/Local/Temp/osm.exe
install -m 755 C:/Users/corsair/AppData/Local/Temp/osm.exe /usr/bin/osm.exe
```

OR

```bash
chmod +x C:\\Users\\corsair\\AppData\\Local\\Temp\\osm.exe
sudo install -m 755 C:\\Users\\corsair\\AppData\\Local\\Temp\\osm.exe /usr/bin/osm.exe
```

## Current Behaviour

The current commands displayed at the end of `ark get osm` will generate the follow errors:
`chmod +x C:\Users\corsair\AppData\Local\Temp/osm.exe`
image

`sudo install -m 755 C:\Users\corsair\AppData\Local\Temp/osm.exe /usr/local/bin/osm.exe`  (and without `sudo`)
image

## Possible Solution

As stated in the "Expected behaviour", the output commands should be updated for Git Bash

## Steps to Reproduce (for bugs)

1. Install Git Bash for Windows: https://gitforwindows.org/
2. Install Arkade for Git Bash: https://github.com/alexellis/arkade#get-arkade
3. Run `ark get osm`
4. Run `chmod +x C:\Users\corsair\AppData\Local\Temp/osm.exe`
5. Run `sudo install -m 755 C:\Users\corsair\AppData\Local\Temp/osm.exe /usr/local/bin/osm.exe`

## Context

Testing Arkade on Git Bash as discussed with @alexellis 

## Your Environment

* What Kubernetes distribution are you using?
Docker Desktop for Windows Kubernetes

```
kubectl version
```
v1.18.6
image

* Operating System and version (e.g. Linux, Windows, MacOS):
Windows 10

```
uname -a

cat /etc/os-release
```
N/A

* What arkade version is this?

```
arkade version
```
0.6.0
image

### PR DESCRIPTION
**Pull request by kadern0**
Signed-off-by: Pablo Caderno <kaderno@gmail.com>

## Motivation and Context

Fixes #169 

- [ ] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

File: cmd/get.go [hunk 1]
```diff
@@ -74,11 +75,18 @@ and provides a fast and easy alternative to a package manager.`,
 		fmt.Printf("Tool written to: %s\n\n", outFilePath)
 
 		if dlMode == get.DownloadTempDir {
-			fmt.Printf(`Run the following to copy to install the tool:
+			if strings.Contains(strings.ToLower(operatingSystem), "mingw") {
+				fmt.Printf(`Run the following to copy to install the tool:
+
+sudo install -m 755 %s /usr/bin/%s
```
Comment (alexellis) [inline cmd/get.go:81]: I don't think that `sudo` is available on Git Bash

**Comment by alexellis** [comment]
@kadern0 we need a rebase on this when you have time please.

**Review by Waterdrips — APPROVED** [review]
lgtm

**Comment by Shikachuu** [comment]
@kadern0 If you still intrested, can you rebase the PR again please?

**Comment by Shikachuu** [comment]
I think we can assume that this PR has been abandoned by the author.
I'll close this for now, but feel free to revisit this in a new PR!

**Pull request closed without merging**